bezier_library/include/bezier_library/bezier_library.hpp

```diff
@@ -157,6 +157,24 @@ class Bezier
   bool
   getTranslationMode();
 
+  /** @brief Set @ref Activate the surfacing mode
+   */
+  void SetSurfacingOn();
+
+  /** @brief Set @ref Deactivate the surfacing mode
+    */
+  void SetSurfacingOff();
+
+  /** @brief Set @ref Set the surfacing mode
+   *  @param[in] surfacing_mode
+   */
+  void SetSurfacing(const bool surfacing_mode);
+
+  /** @brief Get @ref Get the surfacing mode
+    * @param[out] surfacing mode
+    */
+  bool GetSurfacing() const;
+
```

Comment:
Members fonctions names are camel cased with the first character being lower case.

`setSurfacingOn` instead of `SetSurfacingOn`


```
Set @ref Deactivate the surfacing mode
```

This is weird, this is human readable:
```
Deactivate the surfacing mode
```